apps/system/js/app_window_manager.js

```diff
@@ -728,6 +729,10 @@
       }
     },
 
+    _handle_cardviewprepare: function(evt) {
+      this.broadcastMessage('cardviewprepare');
+    },
```

Comment:
do we need this?
Looks like only the `InputWindowManager` needs the event and listens for it on the window (versus all appWindows listening on their element)